src/webhook/Makefile

```diff
@@ -122,6 +122,25 @@ kind-load: ## load the image in the local kind cluster
 .PHONY: kind-deploy
 kind-deploy: docker-build kind-load deploy ## deploy the webhook in the local kind cluster
 
+.PHONY: deploy-cert-manager
+deploy-cert-manager: ## Deploy cert-manager for webhook.
+	curl -fsSL -o cmctl https://github.com/cert-manager/cmctl/releases/latest/download/cmctl_linux_${ARCH}
+	chmod +x cmctl
+	# Deploy cert-manager
+	kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.15.3/cert-manager.yaml
+	# Wait for service to be up
+	kubectl wait --timeout=30s -n cert-manager endpoints/cert-manager --for=jsonpath='{.subsets[0].addresses[0].ip}'
+	kubectl wait --timeout=30s -n cert-manager endpoints/cert-manager-webhook --for=jsonpath='{.subsets[0].addresses[0].ip}'
+	# Wait for few seconds for the cert-manager API to be ready
+	# otherwise you'll hit the error "x509: certificate signed by unknown authority"
+	# Best is to use cmctl - https://cert-manager.io/docs/installation/kubectl/#2-optional-wait-for-cert-manager-webhook-to-be-ready
+	./cmctl check api --wait=2m
```

Comment:
I tested this `deploy-cert-manager` target; worked pretty well. The API was up and running very quickly so maybe we can even reduce the wait time to a few seconds so that, on fail, we don't need to wait 2m.

Reply:
I think wait is the max wait. It'll will exit the moment the api is up.

Reply:
You can try re-running cmctl check api again on a ready system and it'll return immediately. 
Let's leave it at 2m as anyway in CI and otherwise things can be flaky due to network issues and it can take time. 
If you still prefer a lower time, let me know a value and I'll update the PR.

Reply:
no, that's ok. let's keep as is

Reply:
Our messages crossed ;) .. Leaving the time as 2m for now